src/tools/tidy/src/style.rs

```diff
@@ -385,10 +385,10 @@ pub fn check(path: &Path, bad: &mut bool) {
             }
             if filename != "style.rs" {
                 if trimmed.contains("TODO") {
-                    err("TODO is deprecated; use FIXME")
+                    err("TODO is used for tasks that should be done before merging a PR; If you want to leave message in the codebase use FIXME")
                 }
                 if trimmed.contains("//") && trimmed.contains(" XXX") {
-                    err("XXX is deprecated; use FIXME")
+                    err("Instead of XXX use FIXME or TODO")
```

Comment:
Why point the user to TODO just to tell him later that it should not be used?

Reply:
My idea was that TODO is a useful mechanism on it's own, it's just different. I actually wanted to write something like

> Instead of XXX use FIXME (for tasks that can live in the codebase) or TODO (for things that should be done before a PR is merged)

...but got distracted 😅 

This is a bit lengthy, maybe there is a better way to phrase this...

Reply:
A recommended practice is to run `x test tidy` as a pre-push hook, so it seems like having an error telling you to use something just to get another error telling you not to use it seems a bit nonsensical to me.